graphics/jpegoptim/Portfile

```diff
@@ -1,12 +1,11 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
 
-name                jpegoptim
-version             1.4.6
+github.setup        tjko jpegoptim 1.4.7 v
 revision            0
```

Comment:
please add `github.tarball_from archive` and then recalculate the `checksums` as they will have changed